.github/dependabot.yml

```diff
@@ -26,3 +26,13 @@ updates:
       day: "thursday"
       time: "07:00"
       timezone: "Europe/Oslo"
+     
+## Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
```

Comment:
The property order is different from the other entries. Probably doesn't matter, but I'm not sure.

Reply:
That doesn't matter.